### PR TITLE
Fe 349 feed page tx types component and variations

### PR DESCRIPTION
**Pull request by shan8851**
Updated design for current swap transaction and added new tx types from https://uniwhales.atlassian.net/browse/FE-349?atlOrigin=eyJpIjoiYmM3MTA5NTczYjk0NDFkYWE5MDk0MzQwNmQ3ZDMxYTYiLCJwIjoiaiJ9

### CODE REVIEW

File: src/atoms/feedCardItem/dollarPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    height: 156,
+    borderRadius: 8,
+  }}
+  >
+    <p style={{
+      marginBottom: 4,
+      color: '#DD8442',
+    }}
+    >
+      🦐 $100k-$249k volume (#shrimp)
```
Comment (goshatravin) [inline src/atoms/feedCardItem/dollarPopover.tsx:18]: @shan8851 it should be into the text file .

File: src/organisms/feedCard/feedCard.tsx [hunk 2]
```diff
@@ -21,14 +27,15 @@ export const FeedCard = (
...
+    wallet,
   }: FeedCardProps,
 ) => {
   const theme = useTheme() as typeof Theme;
   const firstItem = transactions[0];
   const firstFiveItems = transactions.slice(0, 5);
   const [
     transactionsToDisplay,
-    setTransactionsToDisplay] = useState<TransactionData[]>([firstItem]);
+    setTransactionsToDisplay] = useState<any>([firstItem]);
```
Comment (goshatravin) [inline src/organisms/feedCard/feedCard.tsx:38]: @shan8851  we dont know type for it ?

File: src/organisms/hintsAndHovers/styles.tsx [hunk 3]
```diff
@@ -11,5 +11,6 @@ export const Tooltip = Styled(ReactTooltip)`
   max-width: 296px;
   &.__react_component_tooltip {
     border-radius: 8px;
+    opacity: 1 !important;
```
Comment (goshatravin) [inline src/organisms/hintsAndHovers/styles.tsx:14]: @shan8851 why do we need !important ?

File: src/utils/format.ts [hunk 4]
```diff
@@ -13,3 +13,22 @@ export const formatter = (maximum:boolean) => new Intl.NumberFormat('en-US', {
   // (this suffices for whole numbers, but will print 2500.10 as $2,500.1)
   // maximumFractionDigits: 0, // (causes 2500.99 to be printed as $2,501)
```
Comment (goshatravin) [inline src/utils/format.ts:14]: @shan8851 remove ?

**Review by goshatravin — CHANGES_REQUESTED** [review]

File: src/organisms/hintsAndHovers/styles.tsx (hunk 3 above)
Comment (shan8851) [reply src/organisms/hintsAndHovers/styles.tsx:14]: We don't need this, was testing something out with `hintsAndHovers` and forgot to remove. Thanks for spotting it!

File: src/utils/format.ts (hunk 4 above)
Comment (shan8851) [reply src/utils/format.ts:14]: Screenshot 2022-07-01 at 07 56 10

@goshatravin I think this was your commit I think a couple of months back. I can remove if not needed though, just let me know.

Comment (goshatravin) [reply src/utils/format.ts:14]: @shan8851 ok leave it

**Pull request merged**